packages/components/stories/DatePicker.js

```diff
@@ -83,10 +83,55 @@ storiesOf('DatePicker', module)
 				id="my-date-picker"
 				selectedDateTime={new Date(2018, 4, 13, 12, 30)}
 				onChange={action('onChange')}
+				onFinish={action('onFinish')}
 				name="Datetime"
 			/>
 		</div>
 	))
+	.add('Date picker form mode', () => (
+		<div>
+			<IconsProvider />
+			<h1>DatePicker</h1>
+			<p>
+				Default date picker doesn't include time<br />
+				It comes with classical keyboard gesture.
+			</p>
+			<TestPickerWrapper
+				id="my-date-picker"
+				selectedDateTime={new Date(2018, 4, 13, 12, 30)}
+				onChange={action('onChange')}
+				onFinish={action('onFinish')}
+				name="Datetime"
+				formMode={true}
+			/>
+		</div>
+	))
+	.add('Date picker - time form mode', () => (
+		<div>
+			<IconsProvider />
+			<h1>DatePicker with time</h1>
+			<p>You can require time with a simple "useTime" props.</p>
```

Comment:
Same here, please update the description, the form mode is important here
Perhaps you can merge the 2 stories about form mode to display 2 pickers (with/without time)